server/database/queries/getAvailableRooms.js

```diff
@@ -0,0 +1,16 @@
+const dbconnection = require('../db_connection')
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+
+const availableRooms = (eventDate) => {
+  const sql = {
+    text: 'SELECT room_id FROM events WHERE event_date !=$1',
```

Comment:
as you know, the problem with this query is that you are selecting from `events` table, which contains room IDs only of rooms that have previously been scheduled, not all of the rooms that are in the system. You can either do LEFT JOIN (bit more complicated), or something like
```sql
SELECT room_id FROM rooms WHERE room_id NOT IN 
  (SELECT room_id FROM events WHERE event_date = $1)
```
Notice that I changed the WHERE from `event_date !=$1` to `event_date = $1`. This query will first select all of the rooms that are already booked in the `events` table, and then use the result to exclude those IDs from the the `rooms` table.

Reply:
Done